### PR TITLE
Navbar and Icon

### PR DESCRIPTION
**Pull request by bgtdinh**

### CODE REVIEW

File: components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                        className={classNames(
+                          item.current ? 'text-green' : 'text-grey hover:text-green',
+                          'px-3 py-2 rounded-md text-sm font-semibold'
+                        )}
+                        aria-current={item.current ? 'page' : undefined}
+                      >
+                        {item.name}
+                      </a>
+                    ))}
+                <a key="Join" href="#" className="px-3 py-2 rounded-xl text-lg font-semibold text-green border border-green hover:text-white hover:bg-green">Join</a>
```
Comment (van-hsieh) [inline components/Navbar.jsx]: Hi Brian, I would suggest reducing the padding on the button and may reduce the font size for the button. The Figma mockup has a button with less padding and fits better with the look.

**Review by van-hsieh — COMMENTED** [review]
Great work! Only one small style suggestion.

File: components/Navbar.jsx (hunk 1 above)
Comment (bgtdinh) [reply components/Navbar.jsx]: I changed the styles and moved it to a global style.js so you can reference them.

**Review by van-hsieh — APPROVED** [review]
Thanks for the update. All looks good now!

**Pull request merged**